code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -288,6 +288,7 @@
 	return
 
 /mob/living/simple_animal/attack_hand(mob/living/carbon/human/M)
+	..()

```

Comment:
Simple animals wouldn't call the base /atom/movable attack hand which handles unbuckling